README.rst

```diff
@@ -49,8 +49,19 @@ HistomicsTK can be used in two ways:
   `api-docs <https://digitalslidearchive.github.io/HistomicsTK/api-docs.html>`__
   and `examples <https://digitalslidearchive.github.io/HistomicsTK/examples.html>`__
   for more information.
-
-  This can be installed on Linux via ``pip install histomicstk``.
+  
+  Installation instructions on Linux:
+  
+  *To install HistomicsTK using PyPI*::
+  
+  $ pip install histomicstk
+  
+  *To install HistomicsTK from source*::
+  
+  $ git clone https://github.com/DigitalSlideArchive/HistomicsTK/
+  $ cd HistomicsTK/
+  $ python -m pip install scikit-build ninja cython numpy nimfa
+  $ python setup.py develop
```

Comment:
I would prefer to list this as `python -m pip install -e .` (or we could assume that pip is on the command line and just have `pip install -e .`.

Reply:
OK, done.

Reply:
I tried from line 61-64 on command line of my windows computer but when I executed the 64th line, an error came up. I downloaded Microsoft Visual Studio 2019 but it didn’t help anything. What should I do?

The error says:

********************************************************************************
scikit-build could not get a working generator for your system. Aborting build.

Building windows wheels for Python 3.7 requires Microsoft Visual Studio 2017.
Get it with "Visual Studio 2017":

  https://visualstudio.microsoft.com/vs/

********************************************************************************

Reply:
@satotty I'm not sure how this works on the windows commandline. Ideally, you should be working inside a conda environment to keep things clean. I'd suggest you:

1- Install Anaconda on Widows: https://docs.anaconda.com/anaconda/install/windows/
2- Create a new conda envirnment, say we all it `htk`, and install pip inside it, using something like:
```bash
conda create --name htk
conda install -n htk pip
```
3- Inside the conda command line, activate the environment: `conda activate htk`
4- Follow the same installation instructions provided in the new 'README.rst'